src/Codeception/Test/Test.php

```diff
@@ -88,32 +109,21 @@ final public function run(\PHPUnit\Framework\TestResult $result = null): \PHPUni
         $failedToStart = ReflectionHelper::readPrivateProperty($result, 'lastTestFailed');
 
         if (!$this->ignored && !$failedToStart) {
-            if (null !== $timer) {
-                $timer->start();
-            } else {
-                Timer::start();
-            }
+            $timer->start();
```

Comment:
I find it funny that PHPUnit 9.2 changed Timer from static method to instance, but in PHPUnit 10, PhpCodeCoverage will be executed using static methods instead of injecting it to TestResult.